docs/whats-new/dotnet-docs-2022-08-01.md

```diff
@@ -0,0 +1,108 @@
+---
+title: ".NET docs: What's new for August 2022"
+description: "What's new in the .NET docs for August 2022."
+ms.date: 09/01/2022
+---
+
+# .NET docs: What's new for August 2022
+
+Welcome to what's new in the .NET docs for August 2022. This article lists some of the major changes to docs during this period.
+
+## .NET breaking changes
+
+### New articles
+
+- [BinaryFormatter serialization APIs produce compiler errors](../core/compatibility/core-libraries/7.0/binaryformatter-apis-produce-errors.md)
+- [All assemblies trimmed by default](../core/compatibility/deployment/7.0/trim-all-assemblies.md)
+
+## .NET Framework
+
+### New articles
+
+- [Install .NET Framework on Windows Server 2022](../framework/install/on-server-2022.md)
+- [Retargeting changes for migration from .NET Framework 4.8 to 4.8.1](../framework/migration-guide/retargeting/4.8-4.8.1.md)
+- [Runtime changes for migration from .NET Framework 4.8 to 4.8.1](../framework/migration-guide/runtime/4.8-4.8.1.md)
+
+## .NET fundamentals
+
+### New articles
+
+- [Source generation for custom marshalling](../standard/native-interop/custom-marshalling-source-generation.md)
+- [Title not found in: #30543](../fundamentals/networking/httpclient.md)
+- [BinaryFormatter serialization APIs produce compiler errors](../core/compatibility/core-libraries/7.0/binaryformatter-apis-produce-errors.md)
+- [System.Threading.Channels library](../core/extensions/channels.md)
+- [All assemblies trimmed by default](../core/compatibility/deployment/7.0/trim-all-assemblies.md)
+- [Generic math](../standard/generics/math.md)
+
+### Updated articles
+
+- [Composite formatting](../standard/base-types/composite-formatting.md) - Fix broken build
+- [Install .NET on Windows](../core/install/windows.md) - Add winget install / Update linux tables
+- [Quantifiers in Regular Expressions](../standard/base-types/quantifiers-in-regular-expressions.md) - [Freshness] US-1953530: Refresh top articles by PV - .NET and Languages Content (4)
+
+## Architecture guides
+
+### New articles
+
+- [Accessing remote data](../architecture/maui/accessing-remote-data.md)
+- [Authentication and Authorization](../architecture/maui/authentication-and-authorization.md)
+- [Communicating between loosely coupled components](../architecture/maui/communicating-between-components.md)
+- [Configuration management](../architecture/maui/configuration-management.md)
+- [Dependency injection](../architecture/maui/dependency-injection.md)
+- [Enterprise Application Patterns Using .NET MAUI](../architecture/maui/index.md)
+- [Introduction to .NET MAUI](../architecture/maui/introduction.md)
+- [Containerized Microservices](../architecture/maui/micro-services.md)
+- [Model-View-ViewModel (MVVM)](../architecture/maui/mvvm.md)
+- [Navigation](../architecture/maui/navigation.md)
+- [Purpose](../architecture/maui/preface.md)
+- [Unit testing](../architecture/maui/unit-testing.md)
+- [Validation](../architecture/maui/validation.md)
```

Comment:
Question for the team: Should we coalesce these into one entry for the new MAUI ebook?

Reply:
Yes.